conf.d/fzf.fish

```diff
@@ -49,5 +49,7 @@ function _fzf_uninstall --on-event fzf_uninstall
         echo "fzf.fish key bindings removed"
         set_color normal
     end
+
+    set --erase __fzf_search_vars_cmd
```

Comment:
Out of scope change: while updating `__fzf_search_vars_cmd` I realized I never accounted for it in the uninstaller.